### PR TITLE
osd: correct the func name in execute_ctx() log messages

### PR DESCRIPTION
**Pull request by guzhongyan**
Wrong function name in execute_ctx log

Signed-off-by: Gu Zhongyan <guzhongyan@360.cn>

### CODE REVIEW

File: src/osd/PrimaryLogPG.cc [hunk 1]
```diff
@@ -2970,13 +2970,13 @@ void PrimaryLogPG::execute_ctx(OpContext *ctx)
     ctx->at_version = get_next_version();
     ctx->mtime = m->get_mtime();
 
-    dout(10) << "do_op " << soid << " " << ctx->ops
+    dout(10) << __func__ << soid << " " << ctx->ops
```
Comment (tchaikov) [inline src/osd/PrimaryLogPG.cc]: could have a space after `__func__`.

Comment (guzhongyan) [inline src/osd/PrimaryLogPG.cc]: Done.

**Comment by guzhongyan** [comment]
Anything else I need to do? I see some build error. Can it be merged?

**Comment by tchaikov** [comment]
retest this please (jenkins log expired)

**Comment by guzhongyan** [comment]
So how to trigger the retesting?

**Comment by yuriw** [comment]
test this please

**Comment by tchaikov** [comment]
@guzhongyan yuri just said it.

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2017-03-03_16:16:25-rados-wip-yuri-testing_2017_3_4-distro-basic-smithi/

http://pulpito.ceph.com/yuriw-2017-03-05_23:28:15-rados-wip-yuri-testing_2017_3_4---basic-smithi/
and reproduce
http://pulpito.ceph.com/yuriw-2017-03-05_23:28:45-rados-wip-yuri-testing_2017_3_4---basic-mira/

**Pull request merged**